.github/workflows/frontend-integration-test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     needs: changes
     if: needs.changes.outputs.source_changes == 'true'
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
is this expected?

Reply:
Yes, I have tested twice and it seems to have taken over 30 minutes, so I increased it.